addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -165,7 +165,7 @@ export const editorCommands = {
         const containerFirstChild = document.createElement('fake-element-fc');
         const containerLastChild = document.createElement('fake-element-lc');
 
-        if (content instanceof Node) {
+        if (content instanceof editor.document.defaultView.Node) {
```

Comment:
@dmo-odoo Wait for https://github.com/odoo/odoo/pull/136545 to be fw-ported to master before r+ this one, as this line actually comes from that PR.

Reply:
@dmo-odoo, all good now. You can review it.